docusign/docusign.lua

```diff
@@ -0,0 +1,84 @@
+--[[
+# [DocuSign API](https://docs.docusign.com/esign/guide/usage/quickstart.html) connector.
+
+### Authentication
+
+Authentication use [Authentication](https://admindemo.docusign.com).
+
+## Input parameters
+
+* `username` - application user email used to authorize the call.
+* `password` - user password used to authorize the call
+* `integratorKey` - integration and authenticate with the DocuSign platform
+
+* `baseUrl` - https://demo.docusign.net/restapi/v2/
+* api_endpoint - API endpoint URI.
+* method - optional request method, defauls to `GET`, could be `GET`, `POST`, `PUT`, `DELETE`.
+* post_data - optional JSON string used to post data, e.g. `"documents":
```

Comment:
your json is wrong, there is no { and } brackets around the structure.
there is no final `